app/client/views/home/home.html

```diff
@@ -1,40 +1,42 @@
 <div id="home">
 	<section id="hero">
 		<div class="hero-content center">
-			<div ng-class="{ 'fade-out': fadeOut }">
-				<div class="logo-header-container ui grid mobile only tablet only">
-					<div class="logo-header logo-cog">
-						<img src="../assets/images/logo-cog.svg">
-					</div>
-					<div class="logo-header">
-						<img src="../assets/images/logo-anteater.svg">
-					</div>
-				</div>
-				<span class="hero-title">HackUCI 2018</span>
-				<div class="hero-description-container">
-					<span class="hero-description ui grid computer only tablet only no-margin-right">FEBRUARY 2 – 4,&nbsp;</span>
-					<span class="hero-description ui grid mobile only">FEBRUARY 2 – 4&nbsp;</span>
-					<span class="hero-description">UC IRVINE</span>
-				</div>
-				<div class="hero-button-container">
-					<a ui-sref="app.recruit" class="btn">
-						<i class="fa fa-rocket" aria-hidden="true"></i>
-						TEAM OPENINGS
-					</a>
-					<!-- <a ui-sref="app.expo" class="btn">
-						<i class="fa fa-rocket" aria-hidden="true"></i>
-						LIVE
-					</a> -->
-					<!-- <a ui-sref="app.apply" class="btn" ng-if="regIsOpen">
-						<i class="fa fa-rocket" aria-hidden="true"></i>
-						APPLY
-					</a>
-					<a class="btn btn--coming-soon" ng-if="!regIsOpen">
-						<i class="fa fa-rocket" aria-hidden="true"></i>
-						REGISTRATION CLOSED
-					</a> -->
-				</div>
-			</div>
+            <div ng-class="{ 'fade-out': fadeOut }">
+                <div class="logo-header-container ui grid mobile only tablet only">
+                    <div class="logo-header logo-cog">
+                        <img src="../assets/images/logo-cog.svg">
+                    </div>
+                    <div class="logo-header">
+                        <img src="../assets/images/logo-anteater.svg">
+                    </div>
+                </div>
+                <span class="hero-title">HackUCI 2019</span>
+                <div class="hero-description-container">
+                    <span class="hero-description">COMING SOON&nbsp;</span>
```

Comment:
Could you remove the fixed space after the text?

Reply:
Done and done